packages/manager/cypress/e2e/images/smoke-create-image.spec.ts

```diff
@@ -22,40 +46,18 @@ describe('create image', () => {
     });
 
     // stub incoming response
-    cy.intercept(apiMatcher('images?*'), {
-      results: 0,
-      data: [],
-      page: 1,
-      pages: 1,
-    }).as('getImages');
-    cy.intercept('POST', apiMatcher('images'), mockNewImage).as('createImage');
+    const mockImages = new Array(2).fill(null).map(
+      (item: null, index: number): Image => {
+        return imageFactory.build({
+          label: `Image ${index}`,
+        });
+      }
+    );
+    mockGetImages(mockImages).as('getImages');
```

Comment:
```suggestion
    const mockImages = imageFactory.buildList(2);
    mockGetImages(mockImages).as('getImages');
    
    // Alternatively, we could even use an empty array:
    mockGetImages([]).as('getImages');
```
This could be simplified by using `imageFactory.buildList(2)` since we don't need to manually set the label for the created images for any particular reason.

For some additional context, the `Array(n).fill(null).map()` pattern is really only needed when we need to create an array of objects using a factory and set a specific value on a per item basis, which isn't really the case in this instance. But before we switched to Vite, we were unable to import [`factory.ts`](https://www.npmjs.com/package/factory.ts) from within our Cypress tests (which gives us handy utils like `Factory.each()`), so following this pattern was the next best choice. I think the issue with importing `factory.ts` has probably been resolved since switching to Vite, so I'm thinking we might not need to follow this pattern going forward.